LiteCore/tests/QueryTest.cc

```diff
@@ -46,6 +46,32 @@ static void addNumberedDocs(KeyStore *store) {
     t.commit();
 }
 
+TEST_CASE_METHOD(DataFileTestFixture, "Create/Delete Index", "[Query]") {
+    KeyStore::IndexOptions options { "en", true };
+    store->createIndex("num"_sl, "[[\".num\"]]"_sl, KeyStore::kFullTextIndex, &options);
+    bool gotError = false;
+    try {
```

Comment:
Please use the function `ExpectException` (from LiteCoreTest.hh) — it's simpler, and it sets some state so that the exception can be ignored by the debugger (the Xcode project does this.)